packages/nextra-theme-blog/src/theme-switch.tsx

```diff
@@ -1,11 +1,10 @@
-import React, { useState, useEffect } from 'react'
+import React from 'react'
 import { useTheme } from 'next-themes'
+import { useMounted } from '../../nextra-theme-docs/src/utils/use-mounted'
```

Comment:
does there is a better way to share some code between blog and docs?
- In this PR I need `useMounted` from docs
- In this PR https://github.com/shuding/nextra/pull/525 I need `<MoonIcon />` and `<SunIcon />` from docs
- And in this PR https://github.com/shuding/nextra/pull/509 we can share code-blocks styles to avoid code duplications

Reply:
Since users won't install both blog and docs in one app, I think we can put these outside of these 2 packages, but in a separate folder:

```
packages
  nextra
  nextra-theme-docs
  nextra-theme-blog
  utils
    icons
    hooks
```

Here `utils` is not a package but just a folder. Will that work?

Reply:
currently, this will work, let's wait for a response from @promer94 since he worked on https://github.com/shuding/nextra/pull/569